src/audio-worklet/basic/hello-audio-worklet/main.js

```diff
@@ -3,24 +3,54 @@
 // found in the LICENSE file.
 
 const audioContext = new AudioContext();
+let oscillator; 
+let isPlaying = false; 
 
 const startAudio = async (context) => {
   await context.audioWorklet.addModule('bypass-processor.js');
-  const oscillator = new OscillatorNode(context);
+  oscillator = new OscillatorNode(context);
   const bypasser = new AudioWorkletNode(context, 'bypass-processor');
   oscillator.connect(bypasser).connect(context.destination);
   oscillator.start();
 };
 
+const stopAudio = () => {
```

Comment:
We should leave the rest and simply suspend the context.

Reply:
Thanks for the suggestion. I've modified the stopAudio() function to suspend the audio context instead of stopping and disconnecting the oscillator node. Please review the changes and let me know if they meet the requirements.

Reply:
This also has the same problem with `src/audio-worklet/basic/audio-worklet-node-options/main.js` above. See my comments for the explanation.